packages/connect-iframe/src/index.ts

```diff
@@ -24,17 +24,27 @@ import {
 import { Core, initCore, initTransport } from '@trezor/connect/src/core';
 import { DataManager } from '@trezor/connect/src/data/DataManager';
 import { config } from '@trezor/connect/src/data/config';
-import { initLog } from '@trezor/connect/src/utils/debug';
+import { LogWriter, initLog } from '@trezor/connect/src/utils/debug';
 import { getOrigin } from '@trezor/connect/src/utils/urlUtils';
 import { suggestBridgeInstaller } from '@trezor/connect/src/data/transportInfo';
 import { suggestUdevInstaller } from '@trezor/connect/src/data/udevInfo';
 import { storage, getSystemInfo, getInstallerPackage } from '@trezor/connect-common';
 import { parseConnectSettings, isOriginWhitelisted } from './connectSettings';
+// @ts-expect-error (typescript does not know this is worker constructor, this is done by webpack)
+import LogWorker from './sharedLoggerWorker';
 
 let _core: Core | undefined;
 
+const logWorker = new LogWorker();
+logWorker.port.start();
+
+const logWriter = (): LogWriter => ({
+    add: (message: object) => logWorker.port.postMessage({ type: 'add-log', data: message }),
+});
```

Comment:
maybe these lines could be moved to separate file. here, in packages/connect-iframe/src/index.ts there would stay only import of logWriter? 

I also noticed that logWriter has not been yet passed to some places where we also do logging. for example DeviceList.

Reply:
Right, https://github.com/trezor/trezor-suite/pull/8623/commits/696e9ae916605a632dec3164e25dd3a737afbd9a